hyper/cli.py

```diff
@@ -164,7 +164,12 @@ def set_request_data(args):
     body, headers, params = {}, {}, {}
     for i in args.items:
         if i.sep == SEP_HEADERS:
-            headers[i.key] = i.value
+            # :key:value case
+            if i.key == '':
+                k, v = i.value.split(':')
```

Comment:
While I don't think it's likely we could get a header with a colon in it (the only possible one is `:path`, which *might* be able to have a colon in it), we should defend against that anyway. Let's swap this to `i.value.split(':', 1)`.